scss/style.scss

```diff
@@ -278,9 +278,17 @@
                 vertical-align: top;
             }
         }
+        .lp-json-pollock-layout-carousel-select {
+            position: initial;
+            overflow: auto;
+        }
         .lp-json-pollock-element-text{
             white-space: normal;
         }
+
+        .lp-json-pollock-layout.lp-json-pollock-layout-selected {
+            border-color: $color-green;
```

Comment:
I would check with Brett to make sure the default selected border color should be green. I think we should choose a more brand neutral color for the default.

Reply:
Sure. I'm just waiting for your updates so that we can change the default fallback color later. Thanks!

Reply:
Hi @kelvingraddick, do we have any updates in terms of the color? Or can we use the color that I selected?
Just to be on the same page, I picked this color from the images in the Jira ticket